src/main/java/com/patchworkmc/objectholder/ObjectHolderScanner.java

```diff
@@ -1,29 +1,53 @@
 package com.patchworkmc.objectholder;
 
+import java.util.ArrayList;
+import java.util.List;
 import java.util.Locale;
-import java.util.function.Consumer;
 
 import org.objectweb.asm.AnnotationVisitor;
 import org.objectweb.asm.ClassVisitor;
 import org.objectweb.asm.FieldVisitor;
+import org.objectweb.asm.Label;
+import org.objectweb.asm.MethodVisitor;
 import org.objectweb.asm.Opcodes;
+import org.objectweb.asm.Type;
 
 import com.patchworkmc.Patchwork;
 import com.patchworkmc.annotation.StringAnnotationHandler;
+import com.patchworkmc.util.LambdaVisitors;
 
 public class ObjectHolderScanner extends ClassVisitor {
+	// net.minecraft.util.Registry
+	private static final String REGISTRY = "net/minecraft/class_2378";
```

Comment:
aren't these constants held in that registry util class?

Reply:
Oh yeah, I'll fix that.